e2e/src/utils/utils.po.ts

```diff
@@ -0,0 +1,20 @@
+import { browser, element, by } from 'protractor';
+
+export class Utils {
+  navigateToRoot() {
+    return browser.get(browser.baseUrl) as Promise<any>;
+  }
+
+  navigateToPath(path: string) {
+    const fullUrl = new URL(path, browser.baseUrl);
+    return browser.get(fullUrl.href) as Promise<any>;
+  }
+
+  getCurrentResourcePath() {
+    return browser.getCurrentUrl().then(fullUrl => (new URL(fullUrl)).pathname);
+  }
+
+  getActiveSubNavText() {
+    return element(by.css('.subnav.mtdj__practice_subnav .nav-link.active'));
```

Comment:
I'm not sure you will need this. This was needed for the customer UI which uses some subnav elements.

Reply:
removed